docs/core/entity/climate.md

```diff
@@ -100,7 +100,7 @@ A device's fan can have different states. There are a couple of built-in fan mod
 
 ### Swing modes
 
-The device fan can have different swing modes that it wants the user to know about/control.
+The device fan can have different swing modes that it wants the user to know about/control. There are a couple of built-in presets that will offer translations, but you're also allowed to add custom presets.
```

Comment:
We shouldn't talk about presets since that's another feature.

Reply:
I'm not actually sure we currently allow custom swing modes, even if there are some integrations that do that now.

Reply:
Yeah. It should say swing and not preset. 

I guess we support custom swing modes as otherwise it would have become an enum? It would appear strange if this didn't as every integration/vendor defines it very differently. 
But I have no proof as it wasn't written here it was allowed.

Reply:
Enums is newer than the climate integration. We can't go on that.

Reply:
I did scan through the architecture rep but couldn't find any old issue or discussion that would provide any guidance.

Reply:
Opened discussion: https://github.com/home-assistant/architecture/discussions/1144